### DISCUSSION TITLE
adding option to view website in different languages (english/spanish)

### DISCUSSION THREAD

**Discussion by cvbate** [alshedivat/al-folio#2291] [Q&A] [answered]
TL:DR: how to add the option to view each page in english/spanish (like the dark/light mode butoon)

I speak two languages (english and spanish)- the website  i am currently working on is for my portfolio/resume and i wanted to demonstrate my skills in spanish by having the option to view my website/each page in either English or Spanish.

This is my first website so i am not the most technically advanced in this area, so perhaps its above my skill level, but I am wondering if anyone has had success in adding an additional option (in my mind it would be a button like the dark/light mode) to their website to view each page in an additional language 

thank you!

**Comment by george-gca** [comment, marked as answer]
You might want to start from [this repository](https://github.com/george-gca/multi-language-al-folio) instead.

**Reply by cvbate** [reply]
thank you!
